src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/service/connected/ConnectedDevicesService.java

```diff
@@ -134,11 +135,11 @@ ArpPacketData deserializeArp(Ethernet eth, SwitchId switchId, long cookie) {
     private void handlePacketIn(OfInput input) {
         U64 rawCookie = input.packetInCookie();
 
-        if (rawCookie == null) {
+        if (rawCookie == null || rawCookie.getValue() == UNKNOWN_COOKIE) {
```

Comment:
You should use https://github.com/telstra/open-kilda/blob/develop/src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/model/OfInput.java#L75 here.